errorHandlers/errorHandler.js

```diff
@@ -1,12 +1,12 @@
 exports.handleCustomErrors = (err, req, res, next) => {
   if (err === "article_id out of range") {
-    res.status(400).send({ msg: "bad request" });
+    res.status(404).send({ msg: "404 not found" });
```

Comment:
You may want to make this error handler more flexible, by having something like `if(err.status){...`
If you then do custom Promise.rejects of objects with any status key, like `{ status : 404, msg: 'not found' }`, you can just have one line to send that rather than having to separate your custom errors out by messages you send